client/components/Facilitator/Components/Dashboard/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+// import { Grid } from 'semantic-ui-react';
```

Comment:
I had some grid stuff in here, then I took it out and I left this behind a comment because I knew it was coming back anyway and I was being lazy. Instead of all that, I'm going to update this code to have a grid layout.